src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/service/SearchStaffService.kt

```diff
@@ -0,0 +1,15 @@
+package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.service
+
+import kotlinx.coroutines.flow.Flow
+import kotlinx.coroutines.flow.map
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.controller.dto.StaffDetails
+import uk.gov.justice.digital.hmpps.hmppsstafflookupservice.db.repositories.StaffRepository
+
+@Service
+class SearchStaffService(private val staffRepository: StaffRepository) {
+  suspend fun searchStaff(email: String): Flow<StaffDetails> {
+    return staffRepository.findFirst20ByEmailLikeOrderByEmailAsc("%$email%@%")
```

Comment:
Do we not need a check for at least 2 characters? I think that was a requirement - maybe not